cmd/mycelo/templates.go

```diff
@@ -154,6 +154,10 @@ func (e loadtestEnv) createGenesisConfig(env *env.Environment) (*genesis.Config,
 	genesisConfig.GasPriceMinimum.TargetDensity = fixed.MustNew("0.9999")
 	genesisConfig.GasPriceMinimum.AdjustmentSpeed = fixed.MustNew("0")
 
+	genesisConfig.Hardforks = genesis.HardforkConfig{
```

Comment:
do you want this too?

Reply:
yes. we can't load test donut until we enable replay protection in the load bot (https://github.com/celo-org/celo-blockchain/issues/1488), so this lets us load test until we complete the issue